Packages/com.unity.inputsystem/InputSystem/Plugins/DualShock/DualShockGamepadPS4.cs

```diff
@@ -231,6 +273,115 @@ public class DualShockGamepadPS4 : DualShockGamepad
         ////TODO: move up into base
         public ReadOnlyArray<PS4TouchControl> touches { get; private set; }
 
+        static DualShockGamepadPS4[] devices = new DualShockGamepadPS4[4];
```

Comment:
Should be private s_Devices.

Reply:
Done. 

As a side note I was under the impression that coding standards for C# were changing to match the Microsoft ones, but older code won't change. I'm sure I read it somewhere. Isn't this happening for the InputSystem as its going to be one of the first packages?